hw03_davism5.py

```diff
@@ -0,0 +1,135 @@
+#################################################################################
+# Author: Mauricha Davis
+# Username: davism5
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1GffD-r3WjBjP_JdkNARFkIGOf5sZ6pIk-gUKqU8K7fQ/edit?usp=sharing
+#
+#################################################################################
+
+import turtle
+
+def house(t):
+    for i in range(4):
+        t.forward(300)
+        t.left(90)
+    return t
+
+def roof(t):
+    for i in range(1):
+         t.left(140)
+         t.forward(250)
+         t.left(77)
+         t.forward(260)
+         t.left(143)
+         t.forward(390)
+
+    return t
+
+
+def tree(t):
+    for i in range(2):
+        t.forward(100)
+        t.left(90)
+        t.forward(200)
+        t.left(90)
+
+    return t
+
+def window_door(t):
+    for i in range(2):
+        t.forward(120)
+        t.left(90)
+
+
+    return t
+
+def create_turtle(pensize, color):
+    t_turtle=turtle.Turtle()
+    t_turtle.pensize(pensize)
+    t_turtle.color(color)
+
+    return t_turtle
+
+def main():
+    wn=turtle.Screen()
+    wn.bgcolor('lightblue')
+    wn.screensize(canvwidth=600, canvheight=600)
+
+    t_house= create_turtle(pensize=10, color="hotpink")
+    t_house.penup()
+    t_house.setposition(-280, -280)
+    t_house.pendown()
+
+    t_house.color('hotpink')
+    t_house.begin_fill()
+    for n in range(4):
+        t_house.forward(400)
+        t_house.left(90)
```

Comment:
This is your house() function. Call it here instead:
```
    house(t_house)
```

Reply:
what do you mean call where instead?